pipelinewise/fastsync/postgres_to_bigquery.py

```diff
@@ -0,0 +1,183 @@
+#!/usr/bin/env python3
+import logging
+import os
+import sys
+import time
+from functools import partial
+from argparse import Namespace
+import multiprocessing
+from typing import Union
+
+from datetime import datetime
+from .commons import utils
+from .commons.tap_postgres import FastSyncTapPostgres
+from .commons.target_bigquery import FastSyncTargetBigquery
+
+from .. import utils as pipelinewise_utils
+pipelinewise_utils.QUOTE_CHARACTER = '`'
+
+MAX_NUM="99999999999999999999999999999.999999999"
+
+LOGGER = logging.getLogger(__name__)
+
+REQUIRED_CONFIG_KEYS = {
+    'tap': [
+        'host',
+        'port',
+        'user',
+        'password'
+    ],
+    'target': [
+        'project_id'
+    ]
+}
+
+LOCK = multiprocessing.Lock()
+
+def tap_type_to_target_type(pg_type):
+    """Data type mapping from Postgres to Bigquery"""
+    return {
+        'char':'STRING',
+        'character':'STRING',
+        'varchar':'STRING',
+        'character varying':'STRING',
+        'text':'STRING',
+        'bit': 'BOOL',
```

Comment:
Should we use some other data type instead of `bit` to `BOOL` because if column have bit data type will have data then it will fail to copy the data to bigquery tables?  We had same problem and used `STRING` instead.

Reply:
I won't have much access to Internet for one week but I'll have a look at this after that. The rest of replications use BOOL in this case so I think it makes sense to do the same here. I'll check if this is working for both incremental and fastsync replication and fix accordingly

Reply:
hi! I just did some testing and seems to be working as expected with both full and incremental load in a table with a `BIT` type column. Did you find any issue where that was not working? Could you give me more details so I can have a better look?

Reply:
Does your column with `BIT` data type has value in it? The table which I was trying to sync in bigquery with BIT value failed with error `invalid boolean (011111...)`.

Reply:
oh got it. I was testing it thinking that BIT was only of length one. I'll make the change.
`BIT VARYING` is being converted to number so I think it makes sense to make this one a number as well. Also, BigQuery has a few bit SQL functions that require a number so it would be compatible with that. I'll make the change.

https://github.com/transferwise/pipelinewise/blob/master/pipelinewise/fastsync/postgres_to_snowflake.py#L50
@koszti does this affect the snowflake target as well?

Reply:
It makes sense but tap-postgres transforms bit to boolean in a weird way: https://github.com/transferwise/pipelinewise-tap-postgres/blob/c0eb14ebfe14b10ad208d10c5d1d7f1bf5b8be25/tap_postgres/sync_strategies/logical_replication.py#L242

fastsync postgres-to-bigquery should be in sync with the behaviour of tap-postgres, otherwise the initial sync and the following log-based replications will produce different column types :(

Can we use the existing bit-to-boolean transformation  from tap-postgres for now and later we can consider to change how tap-postgres dealing with bit types? what do you think?

Reply:
sure, that makes sense. I'll check and make sure the bigquery code is doing exactly the same thing. thanks!

Reply:
Yeah, a good idea to make it backward compatible.